migrations/versions/0480_move_heartbeat_templates.py

```diff
@@ -0,0 +1,136 @@
+    WHERE id = '{}'
+"""
+
+NOW = datetime.now(timezone.utc)
+
+def upgrade():
+    heartbeat_service_insert = f"""INSERT INTO services (id, name, created_at, active, count_as_live, message_limit, sms_daily_limit, email_annual_limit, sms_annual_limit, restricted, research_mode, prefix_sms, organisation_type, email_from, created_by_id, version)
+                        VALUES ('{NOTIFY_HEARTBEAT_SERVICE_ID}', 'GCNotify Heartbeat', '{NOW}', True, False, 20000, 20000, 5000000, 5000000, False, False, False, 'central', 'gc.notify.heartbeat.notification.gc',
+                        '{NOTIFY_USER_ID}', 1)
+                    """
+    op.execute(heartbeat_service_insert)
+
+    heartbeat_service_history_insert = f"""INSERT INTO services_history (id, name, created_at, active, count_as_live, message_limit, sms_daily_limit, email_annual_limit, sms_annual_limit, restricted, research_mode, prefix_sms, organisation_type, email_from, created_by_id, version)
+                        VALUES ('{NOTIFY_HEARTBEAT_SERVICE_ID}', 'GCNotify Heartbeat', '{NOW}', True, False, 20000, 20000, 5000000, 5000000, False, False, False, 'central', 'gc.notify.heartbeat.notification.gc',
+                        '{NOTIFY_USER_ID}', 1)
+                     """
+    op.execute(heartbeat_service_history_insert)
+
+    for send_type in ('sms', 'email'):
+        heartbeat_service_permissions_insert = f"""INSERT INTO service_permissions (service_id, permission, created_at) VALUES ('{NOTIFY_HEARTBEAT_SERVICE_ID}', '{send_type}', '{NOW}')"""
+        op.execute(heartbeat_service_permissions_insert)
+
+    # Copy the service permissions from the existing Notify service to the new Heartbeat service.
+    perms_insert = f"""
+        INSERT INTO permissions (id, service_id, user_id, permission, created_at)
+            SELECT uuid_in(md5(random()::text)::cstring), '{NOTIFY_HEARTBEAT_SERVICE_ID}', user_id, permission, '{NOW}'
+              FROM permissions
+             WHERE service_id = '{NOTIFY_SERVICE_ID}'
+    """
+    op.execute(perms_insert)
+
+    # The annual billing is required for new services. Let's copy the annual billing
+    # data from existing Notify service to the new heartbeat service.
+    annual_billing_insert = f"""
+        INSERT INTO annual_billing 
+        (id, service_id, financial_year_start, free_sms_fragment_limit, created_at, updated_at) 
+         SELECT uuid_in(md5(random()::text)::cstring), '{NOTIFY_HEARTBEAT_SERVICE_ID}', financial_year_start, free_sms_fragment_limit, created_at, updated_at 
+           FROM annual_billing
+          WHERE service_id = '{NOTIFY_SERVICE_ID}'
+          ORDER BY financial_year_start DESC
+          LIMIT 1
+    """
+    op.execute(annual_billing_insert)
+
+    user_to_service_insert = f"""
+        INSERT INTO user_to_service 
+            (user_id, service_id)
+            SELECT user_id, '{NOTIFY_HEARTBEAT_SERVICE_ID}'
+              FROM user_to_service
+             WHERE service_id = '{NOTIFY_SERVICE_ID}'
+    """
```

Comment:
The query above is the new change compared to the previous reverted PR. This will carry over all user_to_service associations instead of only the GCNotify system user.